### ISSUE TITLE
Running 3 docker agents in single location – only 1 shows up in WPT server.

### ISSUE THREAD

**Issue by NikitaGl** [WPO-Foundation/wptagent#132]
**My env:**
```
WPT private instance v.17.12 running on Centos 7, PHP 5.6.
3xwptagent v.17.12 running in Docker container.
```

wptagent startup command:
```docker run -d --name wptagent0 -e SERVER_URL="http://wptprivateinstanceip/work/" -e LOCATION="docker-location" -e NAME="Docker Test" -e "SHAPER=none" --cap-add=NET_ADMIN --restart always -m 3g  wptagent```

Sometimes all of them are visible at /install/ page and getLocations.php, but mostly only single agent visible and working. Is there a way to make all of available for testing?

**Comment by pmeenan** [comment]
Try giving the 3 of them different names to see if it helps.  If they all have the same name and come from the same IP then they are going to look like a single agent on the server.

If only one still shows, try running it with -vvvv to enable verbose logging and see what the different agents are displaying.

**Comment by NikitaGl** [comment]
Yes! I discovered it just before noticed your answer. The problem was that I've tried to set NAME environment variable with multiple word value (e.g. NAME="Docker Test Agent"). In fact, I've just removed passing this var at all, so agent may pick container's id/hostname. Thanks for help!

P.S. Unfortunately, I've encountered another issue, that containered agents can run out of disk space, but in current discussion that will be an offtopic. Issue may be closed.
